### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPFAnnotationsStyling_Keyboard: The keyboard users are not able to perform various actions using keyboard

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1471]
Copied from VSO [956777](https://devdiv.visualstudio.com/DevDiv/_queries/edit/956777/)

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: .NET Core 3.0 (WPF)

Application version: VS 2019 Preview 4.7.03190

Windows Version: Windows10

ScreenReader Version: Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link.

Repro Steps:

1) Run exe file for "AnnotationsStyling" from the build app folder provided.
2) Main Window for "AnnotationsStyling" solution will appear.
3) Start screen reader.
4) Navigate to the contents in the main window.

Actual:

When keyboard users navigate using the "Tab" key within the main window they are unable to access the "Next Page",  "Decrease Zoom" and "Increase Zoom" buttons.

The same issue is observed using the both the screen readers i.e. Narrator and NVDA.

Expected:

All the actions that a mouse users is able to perform the same action keyboard and screen reader users  should be available to perform using keyboard.

User Impact:

If the screen reader and keyboard users are not able to interact with the interactive elements using the keyboard then they might not be able to perform all the functions as a mouse user is able to access.

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 1.3.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

MAS 2.1.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
